<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"><title>SWISS-MODEL | Workspace | Model Results | nGL4KP</title><style>
			.sequence { font-size: 12px; font-family:"Courier New", Courier, monospace;}
			h2 {font-size: 1.2em; font-family:Arial; margin:20 0 0 0}
			h3 {font-size: 1.0em; font-family:Arial; margin:0 0 0 0}
			
			.borderedTbl>tbody>tr>td, .borderedTbl th{ 
        text-align:center;
  			border-top:1px solid #DDD;
  			border-left:1px solid #DDD;
      }
      .borderedTbl{
        border-bottom:1px solid #DDD;
        border-right:1px solid #DDD; 
      }

			body {
			width:100%; 
			height:100%; 
			background-color: #E7E8EA;
			margin: 0;
			padding: 0;
			word-wrap: break-word;
			font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
			font-size: 10pt;
			Xcolor: #525252;
			}
			#content{
			padding:20px;
			width:780px;
			margin-left:auto;
			margin-right:auto;
			background-color:#FFF;}
			
			#sib_footer {
			background-color: #333333;
			color: #F7F8FA;
			font-family: Verdana;
			font-size: 10px;
			font-weight: normal;
			height: 20px;
			letter-spacing: 1px;
			
			width:760px;
      margin-left:auto;
      margin-right:auto;
      padding:0px 20px 0px 20px
			}
			
			#sib_footer a {
  			color: #FFFFFF;
  			font-family: Verdana;
  			font-size: 10px;
  			font-weight: normal;
  			letter-spacing: 1px;
			}
			a, a:link, a:visited {
			color: #D6161D;
			text-decoration: none;
			} 	

			.alTbl{ font-family:'Courier New', Courier, monospace; }
	   img{ border:0; }
			
	@media print{
	
	#content{
			padding:20px;
			background-color:#FFF;
		}
	body{ 
	    width:100%;
	    height:100%; 
	    background-color: #FFF;
			padding: 0;
		}	
	#sib_footer{
		display:none;
	}
	}
      
    </style></head><body><div id="content"><table border=0 width="100%"><td valign="middle"><a title="SIB Swiss Institute of Bioinformatics" href="http://www.sib.swiss"><img src="images/sib_ExPASy_logo_small.png"></a></td><td valign="middle"><a title="Biozentrum, University of Basel" href="http://www.biozentrum.unibas.ch"><img src="images/BZ_LOGO_2lines_SW_RGB_POS.png"></a></td><td valign="middle"><div style="font-size:1.9em;">SWISS-MODEL Homology Modelling Report</div></td></tr></table><div style="clear:both"></div><h2>Model Building Report</h2><p>
This document lists the results for the homology modelling project &quot;T451DRAFT_1317&quot; submitted to SWISS-MODEL workspace
on July 11, 2017, 11:10 p.m..The submitted primary amino acid sequence is given in Table T1.</p><p>If you use any results in your research, please cite the relevant publications:</p>
Marco Biasini; Stefan Bienert; Andrew Waterhouse; Konstantin Arnold; Gabriel Studer; Tobias Schmidt; Florian Kiefer; Tiziano Gallo Cassarino; Martino Bertoni; Lorenza Bordoli; Torsten Schwede. (2014). SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information. Nucleic Acids Research (1 July 2014) 42 (W1): W252-W258; doi: 10.1093/nar/gku340.<br>
Arnold, K., Bordoli, L., Kopp, J. and Schwede, T. (2006) The SWISS-MODEL workspace: a web-based environment for protein structure homology modelling. Bioinformatics, 22, 195-201.<br>
Benkert, P., Biasini, M. and Schwede, T. (2011) Toward the estimation of the absolute quality of individual protein structure models. Bioinformatics, 27, 343-350
</p><h2>Results</h2><p>
The SWISS-MODEL template library (SMTL version 2017-07-06, PDB release 2017-06-30) was searched with 
Blast (Altschul et al., 1997) and HHBlits (Remmert, et al., 2011) 

 for evolutionary related structures matching the target sequence in Table T1. For details on the template search, see Materials and Methods. Overall 19 templates were found (Table T2).</p><h2>Models</h2><p>The following models were built (see Materials and Methods "Model Building"):</p><table class="borderedTbl" width="100%" border=0 cellspacing=1 cellpadding=5><thead><tr><th><h3>Model #01</h3></th><th>File</th><th>Built with</th><th>Oligo-State</th><th>Ligands</th><th title="Global Model Quality Estimate">GMQE</th><th>QMEAN</th></tr></thead><tbody><tr><td><a href="model/01/01.png" target="_blank"><img height="60" src="model/01/01.png"></a></td><td><a href="model/01/model.pdb" target="_blank">PDB</a></td><td>
ProMod3 Version 1.0.2.
</td><td>MONOMER</td><td><div style="max-width:350px;">None</div></td><td>0.08</td><td>-1.82</td></tr></table><table width="100%"><tr><td><table cellpadding="0" cellspacing="2"><tr><td width="80">QMEAN</td><td>-1.82</td><td><img height="16" width="100" src="model/01/qmean/energy_profile_plots/qmean4_compact.png"></td></tr><tr><td>C&#946;</td><td>-1.75</td><td><img height="16" width="100" src="model/01/qmean/energy_profile_plots/cbeta_compact.png"></td></tr><tr><td>All Atom</td><td>-1.15</td><td><img height="16" width="100" src="model/01/qmean/energy_profile_plots/all_atom_compact.png"></td></tr><tr><td>Solvation</td><td>-1.72</td><td><img height="16" width="100" src="model/01/qmean/energy_profile_plots/solvation_compact.png"></td></tr><tr><td>Torsion</td><td>-1.19</td><td><img height="16" width="100" src="model/01/qmean/energy_profile_plots/torsion_compact.png"></td></tr></table></td><td><span class="pull-left" style="margin-right:4px;"><a href="model/01/qmean/colored_PDB/local_quality_estimate.png" target="_blank"><img width="180" height="120" src="model/01/qmean/colored_PDB/local_quality_estimate_A.png"></a></td><td><a href="model/01/qmean/colored_PDB/reference_set.png" target="_blank"><img  width="180" height="120" src="model/01/qmean/colored_PDB/reference_set.png"></a></td></tr></table><table class="borderedTbl"><thead><tr><th>Template</th><th>Seq Identity</th><th>Oligo-state</th><th>Found by</th><th>Method</th><th>Resolution</th><th>Seq Similarity</th><th>Range</th><th>Coverage</th><th>Description</th></tr></thead><tbody><tr><td><a href="model/01/templates/4bbw.1.A.pdb" target="_blank" >4bbw.1.A</a></td><td>15.63</td><td>monomer</td><td>HHblits</td><td>X-ray</td><td>2.30&Aring;</td><td>0.27</td><td>
            
            23 - 64
            </td><td>0.36</td><td>SIALIDASE (NEURAMINIDASE)</td></tr></tbody></table> 

    
  The template contained no ligands.
  

<br><pre>Target    MKKLTYTFAGLAFI--IGTFLGANYFFVGSALSDAIKSDSRNVGIEASAHYENYISTSVLVFDLKSVSDSNSPADIFRVF<br>4bbw.1.A  MKRNHYLFTLILLLGCSIFVKASDTVFVHQTQIPILIERQDNVLFYFRLDAKESRMMDEIVLDFGK--------------<br><br>Target    LQYASHMKSKRFDQVVLAHKGSVKFKLKGEYFKTLGEEFEDQNLVYTIRTFPENLYRPDGTQAFGTWTGGIIGVLGEQMN<br>4bbw.1.A  --------------------------------------------------------------------------------<br><br>Target    DFNKFHQDWYIKDMTLATTK<br>4bbw.1.A  --------------------<br><br></pre><br><hr><br><table class="borderedTbl" width="100%" border=0 cellspacing=1 cellpadding=5><thead><tr><th><h3>Model #02</h3></th><th>File</th><th>Built with</th><th>Oligo-State</th><th>Ligands</th><th title="Global Model Quality Estimate">GMQE</th><th>QMEAN</th></tr></thead><tbody><tr><td><a href="model/02/02.png" target="_blank"><img height="60" src="model/02/02.png"></a></td><td><a href="model/02/model.pdb" target="_blank">PDB</a></td><td>
ProMod3 Version 1.0.2.
</td><td>MONOMER 
             
             (matching prediction)
             
          </td><td><div style="max-width:350px;">None</div></td><td>0.07</td><td>-0.97</td></tr></table><table width="100%"><tr><td><table cellpadding="0" cellspacing="2"><tr><td width="80">QMEAN</td><td>-0.97</td><td><img height="16" width="100" src="model/02/qmean/energy_profile_plots/qmean4_compact.png"></td></tr><tr><td>C&#946;</td><td>-2.89</td><td><img height="16" width="100" src="model/02/qmean/energy_profile_plots/cbeta_compact.png"></td></tr><tr><td>All Atom</td><td>0.45</td><td><img height="16" width="100" src="model/02/qmean/energy_profile_plots/all_atom_compact.png"></td></tr><tr><td>Solvation</td><td>0.09</td><td><img height="16" width="100" src="model/02/qmean/energy_profile_plots/solvation_compact.png"></td></tr><tr><td>Torsion</td><td>-0.45</td><td><img height="16" width="100" src="model/02/qmean/energy_profile_plots/torsion_compact.png"></td></tr></table></td><td><span class="pull-left" style="margin-right:4px;"><a href="model/02/qmean/colored_PDB/local_quality_estimate.png" target="_blank"><img width="180" height="120" src="model/02/qmean/colored_PDB/local_quality_estimate_A.png"></a></td><td><a href="model/02/qmean/colored_PDB/reference_set.png" target="_blank"><img  width="180" height="120" src="model/02/qmean/colored_PDB/reference_set.png"></a></td></tr></table><table class="borderedTbl"><thead><tr><th>Template</th><th>Seq Identity</th><th>Oligo-state</th><th>Found by</th><th>Method</th><th>Resolution</th><th>Seq Similarity</th><th>Range</th><th>Coverage</th><th>Description</th></tr></thead><tbody><tr><td><a href="model/02/templates/4wja.1.A.pdb" target="_blank" >4wja.1.A</a></td><td>24.24</td><td>homo-dimer</td><td>HHblits</td><td>X-ray</td><td>2.60&Aring;</td><td>0.30</td><td>
            
            54 - 86
            </td><td>0.19</td><td>Uncharacterized protein C9orf142</td></tr></tbody></table> 

    
  The template contained no ligands.
  

<br><pre>Target    MKKLTYTFAGLAFIIGTFLGANYFFVGSALSDAIKSDSRNVGIEASAHYENYISTSVLVFDLKSVSDSNSPADIFRVFLQ<br>4wja.1.A  -----------------------------------------------------GPSALAFDLSKVPGPEAAPRLRALTLG<br><br>Target    YASHMKSKRFDQVVLAHKGSVKFKLKGEYFKTLGEEFEDQNLVYTIRTFPENLYRPDGTQAFGTWTGGIIGVLGEQMNDF<br>4wja.1.A  LAKRVW--------------------------------------------------------------------------<br><br>Target    NKFHQDWYIKDMTLATTK<br>4wja.1.A  ------------------<br><br></pre><br><hr><br><table class="borderedTbl" width="100%" border=0 cellspacing=1 cellpadding=5><thead><tr><th><h3>Model #03</h3></th><th>File</th><th>Built with</th><th>Oligo-State</th><th>Ligands</th><th title="Global Model Quality Estimate">GMQE</th><th>QMEAN</th></tr></thead><tbody><tr><td><a href="model/03/03.png" target="_blank"><img height="60" src="model/03/03.png"></a></td><td><a href="model/03/model.pdb" target="_blank">PDB</a></td><td>
ProMod3 Version 1.0.2.
</td><td>MONOMER</td><td><div style="max-width:350px;">None</div></td><td>0.04</td><td>-3.70</td></tr></table><table width="100%"><tr><td><table cellpadding="0" cellspacing="2"><tr><td width="80">QMEAN</td><td>-3.70</td><td><img height="16" width="100" src="model/03/qmean/energy_profile_plots/qmean4_compact.png"></td></tr><tr><td>C&#946;</td><td>-3.16</td><td><img height="16" width="100" src="model/03/qmean/energy_profile_plots/cbeta_compact.png"></td></tr><tr><td>All Atom</td><td>-1.41</td><td><img height="16" width="100" src="model/03/qmean/energy_profile_plots/all_atom_compact.png"></td></tr><tr><td>Solvation</td><td>-1.91</td><td><img height="16" width="100" src="model/03/qmean/energy_profile_plots/solvation_compact.png"></td></tr><tr><td>Torsion</td><td>-2.98</td><td><img height="16" width="100" src="model/03/qmean/energy_profile_plots/torsion_compact.png"></td></tr></table></td><td><span class="pull-left" style="margin-right:4px;"><a href="model/03/qmean/colored_PDB/local_quality_estimate.png" target="_blank"><img width="180" height="120" src="model/03/qmean/colored_PDB/local_quality_estimate_A.png"></a></td><td><a href="model/03/qmean/colored_PDB/reference_set.png" target="_blank"><img  width="180" height="120" src="model/03/qmean/colored_PDB/reference_set.png"></a></td></tr></table><table class="borderedTbl"><thead><tr><th>Template</th><th>Seq Identity</th><th>Oligo-state</th><th>Found by</th><th>Method</th><th>Resolution</th><th>Seq Similarity</th><th>Range</th><th>Coverage</th><th>Description</th></tr></thead><tbody><tr><td><a href="model/03/templates/4acz.2.A.pdb" target="_blank" >4acz.2.A</a></td><td>25.00</td><td>monomer</td><td>HHblits</td><td>X-ray</td><td>1.99&Aring;</td><td>0.33</td><td>
            
            124 - 151
            </td><td>0.16</td><td>ENDO-ALPHA-MANNOSIDASE</td></tr></tbody></table><table width="100%" class="borderedTbl"><thead><tr><th>Ligand</th><th>Added to Model</th><th>Description</th></tr></thead><tbody><tr><td>GOL</td><td>&#10005; -
        Not biologically relevant.
      
      </td><td><div style="max-width:650px;">GLYCEROL</div></td></tr><tr><td>GOL</td><td>&#10005; -
        Not biologically relevant.
      
      </td><td><div style="max-width:650px;">GLYCEROL</div></td></tr></tbody></table><br><pre>Target    MKKLTYTFAGLAFIIGTFLGANYFFVGSALSDAIKSDSRNVGIEASAHYENYISTSVLVFDLKSVSDSNSPADIFRVFLQ<br>4acz.2.A  --------------------------------------------------------------------------------<br><br>Target    YASHMKSKRFDQVVLAHKGSVKFKLKGEYFKTLGEEFEDQNLVYTIRTFP--ENLYRPDGTQAFGTWTGGIIGVLGEQMN<br>4acz.2.A  -------------------------------------------YIVDTYGDHPAFYRTHGKPLFFIYDSYLIK-------<br><br>Target    DFNKFHQDWYIKDMTLATTK<br>4acz.2.A  --------------------<br><br></pre><br><hr><br><h2>Materials and Methods</h2><h2>Template Search</h2><p>Template search with Blast and HHBlits
 has been performed against the SWISS-MODEL template library (SMTL, last update: 2017-07-06, last included PDB release: 2017-06-30).</p><p>
The target sequence was searched with BLAST (Altschul et al., 1997) against the primary amino acid sequence contained in the SMTL.



</p><p>
An initial HHblits profile has been built using the procedure outlined in (Remmert, et al., 2011), followed by 1 iteration of HHblits against NR20. The obtained profile has then be searched against all profiles of the SMTL. A total of 20 templates were found.</p><h2>Template Selection</h2><p>For each identified template, the template's quality has been predicted from features of the target-template alignment.
 The templates with the highest quality have then been selected for model building.</p><h2>Model Building</h2><p>
Models are built based on the target-template alignment using ProMod3. Coordinates which are conserved between the target and the template are copied from the template to the model. Insertions and deletions are remodelled using a fragment library. Side chains are then rebuilt. Finally, the geometry of the resulting model is regularized by using a force field. In case loop modelling with ProMod3 fails, an alternative model is built with PROMOD-II (Guex, et al., 1997).
</p><h2>Model Quality Estimation</h2><p>
The global and per-residue model quality has been assessed using the QMEAN scoring function (Benkert, et al., 2011) . For improved performance, weights of the individual QMEAN terms have been trained specifically for SWISS-MODEL. 
</p><h2>Ligand Modelling</h2><p>
Ligands present in the template structure are transferred by homology to the model when the following criteria are met (Gallo -Casserino, to be published): (a) The ligands are annotated as biologically relevant in the template library, (b) the ligand is in contact with the model, (c) the ligand is not clashing with the protein, (d) the residues in contact with the ligand are conserved between the target and the template. If any of these four criteria is not satisfied, a certain ligand will not be included in the model. The model summary includes information on why and which ligand has not been included.</p><h2>Oligomeric State Conservation</h2><p>
Homo-oligomeric structure of the target protein is predicted based on the analysis of pairwise interfaces of the identified template structures. For each relevant interface between polypeptide chains (interfaces with more than 10 residue-residue interactions), the QscoreOligomer (Mariani et al., 2011) is predicted from features such as similarity to target and frequency of observing this interface in the identified templates (Kiefer, Bertoni, Biasini, to be published). The prediction is performed with a random forest regressor using these features as input parameters to predict the probability of conservation for each interface. The QscoreOligomer of the whole complex is then calculated as the weight-averaged QscoreOligomer of the interfaces. The oligomeric state of the target is predicted to be the same as in the template when QscoreOligomer is predicted to be higher or equal to 0.5.</p><h2>References</h2><p>
Altschul, S.F., Madden, T.L., Schaffer, A.A., Zhang, J., Zhang, Z., Miller, W. and Lipman, D.J. (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res, 25, 3389-3402.<br>
Remmert, M., Biegert, A., Hauser, A. and Soding, J. (2012) HHblits: lightning-fast iterative protein sequence searching by HMM-HMM alignment. Nat Methods, 9, 173-175.<br>
Guex, N. and Peitsch, M.C. (1997) SWISS-MODEL and the Swiss-PdbViewer: an environment for comparative protein modeling. Electrophoresis, 18, 2714-2723.<br>
Sali, A. and Blundell, T.L. (1993) Comparative protein modelling by satisfaction of spatial restraints. J Mol Biol, 234, 779-815.<br>
Benkert, P., Biasini, M. and Schwede, T. (2011) Toward the estimation of the absolute quality of individual protein structure models. Bioinformatics, 27, 343-350.<br>
Mariani, V., Kiefer, F., Schmidt, T., Haas, J. and Schwede, T. (2011) Assessment of template based protein structure predictions in CASP9. Proteins, 79 Suppl 10, 37-58.



<h2>Table T1:</h2><p>Primary amino acid sequence for which templates were searched and models were built.</p><div class="sequence">MKKLTYTFAGLAFIIGTFLGANYFFVGSALSDAIKSDSRNVGIEASAHYENYISTSVLVFDLKSVSDSNSPADIFRVFLQYASHMKSKRFDQVVLAHKGS<br>VKFKLKGEYFKTLGEEFEDQNLVYTIRTFPENLYRPDGTQAFGTWTGGIIGVLGEQMNDFNKFHQDWYIKDMTLATTK<br></div><h2>Table T2:</h2><table class="borderedTbl"><thead><tr><th>Template</th><th>Seq Identity</th><th>Oligo-state</th><th>Found by</th><th>Method</th><th>Resolution</th><th>Seq Similarity</th><th>Coverage</th><th>Description</th></tr></thead><tbody><tr><td>4bbw.1.A</td><td>15.63</td><td>monomer</td><td>HHblits</td><td>X-ray</td><td>2.30&Aring;</td><td>0.27</td><td>0.36</td><td>SIALIDASE (NEURAMINIDASE)</td></tr><tr><td>3wtf.1.A</td><td>26.47</td><td>homo-dimer</td><td>HHblits</td><td>X-ray</td><td>3.45&Aring;</td><td>0.31</td><td>0.19</td><td>Uncharacterized protein C9orf142</td></tr><tr><td>3wtf.1.B</td><td>26.47</td><td>homo-dimer</td><td>HHblits</td><td>X-ray</td><td>3.45&Aring;</td><td>0.31</td><td>0.19</td><td>Uncharacterized protein C9orf142</td></tr><tr><td>4wja.1.A</td><td>24.24</td><td>homo-dimer</td><td>HHblits</td><td>X-ray</td><td>2.60&Aring;</td><td>0.30</td><td>0.19</td><td>Uncharacterized protein C9orf142</td></tr><tr><td>4wja.1.B</td><td>24.24</td><td>homo-dimer</td><td>HHblits</td><td>X-ray</td><td>2.60&Aring;</td><td>0.30</td><td>0.19</td><td>Uncharacterized protein C9orf142</td></tr><tr><td>3wtd.1.A</td><td>24.24</td><td>homo-dimer</td><td>HHblits</td><td>X-ray</td><td>2.35&Aring;</td><td>0.30</td><td>0.19</td><td>Uncharacterized protein C9orf142</td></tr><tr><td>3wtd.1.B</td><td>24.24</td><td>homo-dimer</td><td>HHblits</td><td>X-ray</td><td>2.35&Aring;</td><td>0.30</td><td>0.19</td><td>Uncharacterized protein C9orf142</td></tr><tr><td>4acz.2.A</td><td>25.00</td><td>monomer</td><td>HHblits</td><td>X-ray</td><td>1.99&Aring;</td><td>0.33</td><td>0.16</td><td>ENDO-ALPHA-MANNOSIDASE</td></tr><tr><td>5ibo.1.A</td><td>30.77</td><td>monomer</td><td>HHblits</td><td>X-ray</td><td>1.95&Aring;</td><td>0.36</td><td>0.15</td><td>Oplophorus-luciferin 2-monooxygenase catalytic subunit</td></tr><tr><td>5ibo.2.A</td><td>30.77</td><td>monomer</td><td>HHblits</td><td>X-ray</td><td>1.95&Aring;</td><td>0.36</td><td>0.15</td><td>Oplophorus-luciferin 2-monooxygenase catalytic subunit</td></tr><tr><td>5b0u.1.A</td><td>30.77</td><td>monomer</td><td>HHblits</td><td>X-ray</td><td>1.71&Aring;</td><td>0.36</td><td>0.15</td><td>Oplophorus-luciferin 2-monooxygenase catalytic subunit</td></tr><tr><td>5b0u.2.A</td><td>30.77</td><td>monomer</td><td>HHblits</td><td>X-ray</td><td>1.71&Aring;</td><td>0.36</td><td>0.15</td><td>Oplophorus-luciferin 2-monooxygenase catalytic subunit</td></tr><tr><td>3j7q.1.k</td><td>17.86</td><td>hetero-oligomer</td><td>HHblits</td><td>EM</td><td>3.50&Aring;</td><td>0.30</td><td>0.16</td><td>Sec61 gamma subunit</td></tr><tr><td>4cg5.1.B</td><td>17.86</td><td>hetero-oligomer</td><td>HHblits</td><td>EM</td><td>7.40&Aring;</td><td>0.30</td><td>0.16</td><td>PROTEIN TRANSPORT PROTEIN SEC61 SUBUNIT GAMMA</td></tr><tr><td>4cg7.1.B</td><td>17.86</td><td>hetero-oligomer</td><td>HHblits</td><td>EM</td><td>6.90&Aring;</td><td>0.30</td><td>0.16</td><td>PROTEIN TRANSPORT PROTEIN SEC61 SUBUNIT GAMMA</td></tr><tr><td>4cg6.1.B</td><td>17.86</td><td>hetero-oligomer</td><td>HHblits</td><td>EM</td><td>7.80&Aring;</td><td>0.30</td><td>0.16</td><td>PROTEIN TRANSPORT PROTEIN SEC61 SUBUNIT GAMMA</td></tr><tr><td>2wwb.1.B</td><td>17.86</td><td>hetero-oligomer</td><td>HHblits</td><td>EM</td><td>6.48&Aring;</td><td>0.30</td><td>0.16</td><td>PROTEIN TRANSPORT PROTEIN SEC61 SUBUNIT GAMMA</td></tr><tr><td>4v6m.1.3</td><td>8.33</td><td>hetero-oligomer</td><td>HHblits</td><td>EM</td><td>NA</td><td>0.24</td><td>0.13</td><td>Preprotein translocase secE subunit</td></tr><tr><td>5abb.1.B</td><td>8.33</td><td>hetero-oligomer</td><td>HHblits</td><td>EM</td><td>8.00&Aring;</td><td>0.24</td><td>0.13</td><td>PROTEIN TRANSLOCASE SUBUNIT SECE</td></tr></tbody></table></div><div id="sib_footer"><div id="sib_footer_content"><a href="http://www.sib.swiss">Swiss Institute of Bioinformatics</a><a href="mailto:help-swissmodel@unibas.ch" style="margin-left:16px;">Contact Us</a><div id="sib_footer_right" style="float:right"></div></div></div></body></html>
